omniduct/duct.py

```diff
@@ -34,6 +34,8 @@ def __init__(cls, name, bases, dct):
                     cls._protocols[key] = cls
 
     def _for_protocol(cls, key):
+        if key not in cls._protocols:
+            raise KeyError('duct protocol {} is not registered'.format(key))
```

Comment:
For consistency in formatting with other logging messages, can we change this to:
"Missing `Duct` implementation for protocol: '{}'."